bentoml/adapters/multi_image_input.py

```diff
@@ -11,21 +11,32 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import argparse
+import pathlib
+from typing import Iterable
+
+from requests_toolbelt.multipart import decoder
+from werkzeug import Request
+from werkzeug.utils import secure_filename
 
 from bentoml.adapters.base_input import BaseInputAdapter
-from bentoml.adapters.image_input import get_default_accept_image_formats
+from bentoml.adapters.image_input import (
+    get_default_accept_image_formats,
+    verify_image_format_or_raise,
+    _import_imageio_imread,
+)
+from bentoml.exceptions import BadInput
+from bentoml.marshal.utils import SimpleRequest, SimpleResponse
+
+imread = _import_imageio_imread()
```

Comment:
This will raise a MissingDependencyException for BentoML users that does not have `imageio` installed. The existing pattern was for avoiding that issue. `imageio` to BentoML is an optional dependency, only users who need ImageInput or MultiImageInput are required to install it.

Reply:
Surfacing this comment again, I think this still needs to change, otherwise, it will break for users who do not have imageio installed in their environment.  Too bad there isn't a test to cover this yet, because `imageio` is in the test dependencies list. But one thing you can try is first uninstall `imageio`, and then run `import bentoml`, you should see an import error.

Reply:
Currently, I am just using the same pattern (and function) as `ImageInput`, which is to raise a `bentoml.exceptions.MissingDepedencyException` if `imageio` is not present. Would you prefer different behaviour if the module is missing?

Reply:
The change I was requesting was to move this line to inside the `__init__` method of the MultiImageInput adapter class. The reason as I explained above, with the current implementation, users who did not have imageio installed, will run into an error when `import bentoml`

Reply:
actually - I just submitted a PR that added lazy loading, this won't be a problem anymore. Let's keep it this way as you have implemented and we can even migrate the others away from the `self.imageio` pattern later on  https://github.com/bentoml/BentoML/pull/878